src/components/Edit/NatureOfBusiness.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div id="nature-of-business-info">
+    <NatureOfBusinessShared
+      v-if="haveNaics"
+      :showErrors="false"
+      :naics="getCurrentNaics"
+      :naics-services="naicsServices"
+      :hasNaicsChanges="hasNatureOfBusinessChanged"
+      @undoNaics="setNaics(originalNaics)"
+      @setNaics="setNaics($event)"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { Action, Getter } from 'vuex-class'
+import { Component, Vue } from 'vue-property-decorator'
+import { NaicsServices } from '@/services'
+import { ActionBindingIF, BusinessInformationIF, EntitySnapshotIF, NaicsIF } from '@/interfaces'
+import { NatureOfBusiness as NatureOfBusinessShared } from '@bcrs-shared-components/nature-of-business'
+
+@Component({
+  components: {
+    NatureOfBusinessShared
+  }
+})
+export default class NatureOfBusiness extends Vue {
+  private naicsServices = NaicsServices
+
+  @Getter getBusinessInformation!: BusinessInformationIF
+  @Getter getCurrentNaics!: NaicsIF
+  @Getter getEntitySnapshot!: EntitySnapshotIF
+  @Getter hasNatureOfBusinessChanged!: boolean
+
+  @Action setNaics!: ActionBindingIF
+
+  /** The naics data on record for the business. */
+  private get originalNaics (): NaicsIF {
+    return {
+      naicsCode: this.getEntitySnapshot.businessInfo.naicsCode,
+      naicsDescription: this.getEntitySnapshot.businessInfo.naicsDescription
+    }
+  }
+
+  private get haveNaics (): boolean {
+    return (!!this.getBusinessInformation.naicsCode && !!this.getBusinessInformation.naicsDescription)
+  }
+}
+</script>
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+::v-deep {
+  #nob-change-btn {
+    padding-right: 2px;
+  }
+
+  #nob-more-actions {
+   margin-right: -14px;
+  }
+}
```

Comment:
Ayup. I don't think we'll get away from these kinds of differences. I wish we didn't have to do this but this is fine.

Reply:
Agreed. 
When adding the change elements to NOB i tried to keep it generic and allow this local integration to house these "adjustments" (as much as i wish this didn't exist either 😩 )

Reply:
With your change to this shared component, it looks OK without this margin (eg, in Storybook), so this CSS override is to match the local app, and I think that's as good as it gets.

I blame the local app for weird styling. (Just kidding, haha.)

Reply:
hahah i agree!